### ISSUE TITLE
Change author on extension stores from griffpatch

### ISSUE THREAD

**Issue by Hans5958** [ScratchAddons/ScratchAddons#6740]
Both AMO and Edge Store lists "griffpatch" as the author of the extension. I propose changing it looking at this facts.

a. Griffpatch does not contribute significantly these days, and acts as an advisor.
b. Scratch Addons has been used by more than 200k users. It's safe to assume that we already quite established on the ecosystem now, and we can stop piggybacking his name.
c. The contributors as a whole deserves the credit, not just griffpatch.

I'm thinking to changing it to "Scratch Addons contributors". Multiple authors can be added on AMO, but I don't think this works well for other stores.

This has been discussed before, but I just want to formalize it as an issue.

**Comment by WorldLanguages** [comment]
Last time I checked, it wasn't possible to change it in the Edge Store.
But in the Mozilla Firefox store, I could rename my account to "Scratch Addons contributors". It would be nice if all stores allowed the publisher to be a link (scratchaddons.com) just like the Chrome Web Store.

**Comment by WorldLanguages** [comment]
> Last time I checked, it wasn't possible to change it in the Edge Store.

I mean, it may be possible to change it if I ask griffpatch for help, but I can't do it by myself as far as I know.

**Comment by Hans5958** [comment]
> I mean, it may be possible to change it if I ask griffpatch for help, but I can't do it by myself as far as I know.

Why not transfer it to an account of our custody?

**Comment by Hans5958** [comment]
Note that the author is now shown on the new CWS. It says "scratch-addons" as the developer, currently.

**Comment by WorldLanguages** [comment]
@Hans5958 Right. It seems like that field accepts spaces, so it could potentially be changed to `Scratch Addons contributors`.

**Comment by WorldLanguages** [comment]
Looks like Chrome acknowledged showing the developer name when there's a website link was a breaking change, so "Scratch Addons contributors" is no longer visible in the Chrome Web Store

**Comment by Hans5958** [comment]
> Looks like Chrome acknowledged showing the developer name when there's a website link was a breaking change, so "Scratch Addons contributors" is no longer visible in the Chrome Web Store

Are you sure that it is the case? This is on [Google Translate](https://chromewebstore.google.com/detail/google-translate/aapbdbdomjkkjkaonfhkkikfgjllcleb).

![image](https://github.com/ScratchAddons/ScratchAddons/assets/11584103/f4f1c5d9-ffae-495e-b75d-45136a80c8a6)

**Comment by WorldLanguages** [comment]
> Are you sure that it is the case? This is on [Google Translate](https://chromewebstore.google.com/detail/google-translate/aapbdbdomjkkjkaonfhkkikfgjllcleb).

The author in that case is not a link. Just like the old CWS, the new CWS displays either the website link or the author name, never both at the same time.
In the case of our store item, the author was always a link to `scratchaddons.com`.

**Comment by Hans5958** [comment]
Huh, but you can still give a website link anyway, even if you put a name, so why not do that?

**Comment by WorldLanguages** [comment]
Looks like the 2 cases you choose are specific extensions that need  to provide a legal name and physical address because there's a company behind them or because they provide subscriptions/purchases

**Comment by OneShot-Niko** [comment]
<sub>bump?</sub>
Shouldn't this be labeled https://github.com/ScratchAddons/ScratchAddons/labels/type%3A%20enhancement instead of https://github.com/ScratchAddons/ScratchAddons/labels/type%3A%20bug ?
